### DISCUSSION TITLE
Página de Vacunación terminado ya al 100%.

### DISCUSSION THREAD

**Discussion by OmarVenturaP** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#317] [Show and tell]
Les muestro mi página terminada, la verdad me fue mejor de lo que esperaba. Pero si creo que salió bien.
Aun me falta el responsive, pero eso ya lo hare mas despacio.

Espero sus comentarios. Saludos

<b>UPDATE:</b>  Les dejo el link de la <a href="https://omarventurap.github.io/Practica-Frontend-Sem3-CSS/" target="blank">Página de Vacunación</a> ya con estilo responsive. 
Además les dejo el <a href="https://github.com/OmarVenturaP/Practica-Frontend-Sem3-CSS" target="blank"> código en mi repositorio</a>, por si alguno quiere darle una analizada y darme un Feedback o tomar algo que les pueda ayudar.

![landin-page-vaccine-complete](https://user-images.githubusercontent.com/99059639/157387557-20b7b57b-8791-4f7b-a361-786944d54772.png)

**Comment by JuanSalvador73** [comment]
Felicidades Sr, ahí luego nos pasa unos tips :D

**Reply by OmarVenturaP** [reply]
Gracias. lo que se necesite y si está en mis posibilidades con gusto. La verdad me emocionó mucho hacer esta práctica, pero si necesita mucha paciencia.

**Reply by lunasaurio** [reply]
Como lo hiciste? En boostrap o algo así o tu mismo desde cero? es que lo estoy tratando de hacer yo mismo y me esta costando mucho trabajo por que no se como igualar las cosas, estoy buscando consejos

**Reply by Ingrzs** [reply]
@lunasaurio en te cuesta trabajo?
Unos consejos que son basicos pero importantes
1-Aprende la jerarquía

2- aprende las diferentes position
Asi como cuando usar top, left, right,bottom

3- Aprende el margin y el padding

3-display

4 y flexbox

Ya lo demas solo es color,tipo de letra decoración

**Comment by Ingrzs** [comment]
Los iconos los usaste de awesome o flaticon?

**Reply by OmarVenturaP** [reply]
De Font Awsome. De allí jale todos los iconos

**Reply by Ingrzs** [reply]
No encontré los amarillos 😅, o lo hiciste con css el color?

**Reply by OmarVenturaP** [reply]
Si, cambias de color con CSS. Yo le puse un GOLD.

**Reply by SujeiRodriguez** [reply]
:O que buenos tips pasan por aqui

**Reply by Ingrzs** [reply]
@OmarVenturaP estoy modificando los iconos pero en css no me sale los efectos usando el nombre de su clase , si añado una clase mas al icono pues este ya no se muestra en la pagina porque obviamente este tiene su nombre desde el servidor y modificarlo implica que el icono cambia de nombre y ya no se puede hacer nada...

En este caso si modifico el icono desde html si se puede pero no es la opción mas correcta no? En este caso como lo modificaste tu?
Le añadiste un id al icono pero como si ya ahi estarías modificando la clase 😅.
Espero tus consejos... Gracias

**Reply by OmarVenturaP** [reply]
La clase la puse adelante de la que trae. De todas maneras con ID se puede. Aunque recuerda que para modificarlo necesitas invocar el atributo. Puedes intentarlo con la clase y el atributo. Ej. 

.icon_title i {

}

Aquí x ejemplo puse clase y aparte el tipo de atributo "i". Ya allí le agregas los estilos necesarios.

 Intenta y me comentas

**Reply by Ingrzs** [reply]
Si sale, de echo investigue y mejor le puse id,  la cuestión es que tenia dudas cuando y como usar doble "llamado" en css

Ejemplo
Id, clase (con comas)
{Ahi implica que aplicara a cada valor por separado}

Si es clase y elementos u otro

Ejemplo

.clase p (sin comas)
{
Estas siendo específico a solo uno dentro de otro
}


Además veo que tú pusiste
Clase y elemento

Y ami no me salia porque ponía
i .clasetitle (al revés)

aunque literal seria lo mismo no?

Clase i
i clase

Saludos gracias por la respuesta

**Reply by OmarVenturaP** [reply]
Si pues. Es considero que quizás por el tipo de valor. Ya vez que aquí si manejan la especificidad. Puede ser que por eso no jale a veces.

**Reply by OmarVenturaP** [reply]
Pero que bueno que ya te agarro. Saludos igual y éxito.

**Reply by Ingrzs** [reply]
@OmarVenturaP hoy aprendi mucho y eso que solo es css ,no e usado nada de bostrap pero dan ganas de usarlo porque dicen que ahorra tiempo y cuando buscas chamba piden exp con bostrap,gracias por tu amabilidad de tener tiempo, éxitos.

**Reply by OmarVenturaP** [reply]
Así es amigazo. También yo le estoy hechando a Bootstrap no creas,  xk la neta si te aliviana bastante...
de todas maneras aquí andamos, cualquier cosa que se necesite con gusto estamos a la orden.

Éxito y nos vemos en el camino. 
Saludos

**Comment by juanrdzv** [comment]
Super bien! Aprovechando ojala me puedas dar una orientada.

En teoria ya termine la mia pero tengo dos problemas.
1.- Cuando hago mas chica la pantalla muchas cosas se mueven de su lugar y ya no se ajustan, como podria resolverlo?

2.- en el area de emergency contact hay 4 cuadros y yo los quise separar por areas proporcionales y solo se qudan pegado. algo que pueda hacer para que queden a la medida y separados ? 

Gracias !! @OmarVenturaP

**Reply by OmarVenturaP** [reply]
Buen día, tratare de darte algunos tips que me han ayudado, ya que la verdad también soy un novato. Apenas voy empezando con esto de la programación. 

Pero te se decir que en el caso del responsive, a mi me sirve mucho el utilizar porcentajes en vez de medidas, para que al ir reduciendo la pantalla se vaya ajustando. o a su vez, utilizar @media para poder ajustar dependiendo del máximo de la pantalla. (Yo apenas estoy trabajando en el responsive)

En el caso de los cuadros de emergency call yo ocupo "flex-wrap: wrap" y tambien "display: flex" alli con ese se ajustan al tamaño del padre y se distribuyen proporcionalmente. solo dale al padre un tamaño que sea proporcional a la mitad de tu página, para que distribuyas la imagen tambien.

**Reply by juanrdzv** [reply]
va que va! Lo voy a calar ahorita y mejorarla en ese aspecto.
Te agradezco Omar, gusto saludarte !

**Reply by OmarVenturaP** [reply]
Quedamos a la orden. Cualquier cosa si está en mis posibilidades adelante. Saludos

**Comment by AlejandroRD98** [comment]
Muy buena página, te quedo igual a la original. No se sí podrias orientrame de como hacer que la barra superior quede encima de la primerra sección y de donde conseguir las imagenes para cada parte, porfavor.

Grcaias @OmarVenturaP.

**Reply by OmarVenturaP** [reply]
Gracias compañero. Pues al header puedes ponerle un "position: fixed" y queda en la parte superior, nada mas dale una opacidad para que tenga el mismo efecto. Y las imágenes, vi que un compañero las publicó en teams, te comparto el link: 
https://teams.microsoft.com/l/message/19:1a776ca2c00947bda02246a2cbffce74@thread.tacv2/1646641620093?tenantId=4ae54b05-b77e-4224-aef1-8661422e0816&groupId=d38ceaac-3599-4f30-94ba-740ccb530cb0&parentMessageId=1646446326864&teamName=Misi%C3%B3n%20Frontend%20-%20JS&channelName=Semana%203%20-%20Volando%20con%20estilo%20%F0%9F%9B%B0%EF%B8%8F%F0%9F%8E%A8&createdTime=1646641620093

Yo en mi caso las busque con Google Lens. Me salieron las que eran parecidas y esas ocupé

**Reply by AlejandroRD98** [reply]
Muchas gracias!!!
Otra cosa, nosé si me pudieras guiar en como hacer la parte de contcato de emergencía estoy teniendo problemas para hacer la cuadricula así, porfavor

**Reply by OmarVenturaP** [reply]
Te recomiendo colocarlos dentro de un div. Ya que ese será tu elemento padre. Una vez allí le das medida a tu elemento padre y le das la propiedad "flex-wrap: wrap" y display: Flex", para que te los distribuya.

Ya a los elementos que tienes dentro nada más les ajustas los tamaños y márgenes para que te queden centrados y ajustados a tu página.
Es de paciencia, pero te sugiero que lo hagas solo con un elemento, ya nada más les pasas las propiedades a los demás y te queda igual.

**Comment by LeonardoRosalesG** [comment]
Una pregunta, ¿Qué comandos usaste en la parte de "Preparations..." para abrir las recomendaciones?

**Reply by OmarVenturaP** [reply]
Allí use código de Javascript. Y una clase en CSS para ampliar el cuadro de texto al implementar JS.

**Reply by OmarVenturaP** [reply]
Si gustas en un ratito más te lo comparto.

**Reply by Ingrzs** [reply]
Tal ves con una select list, Solo al nombre de la lista ponle un icono y no un nombre, por supuesto la mejor manera sigue siendo con js

**Reply by JonathanJRodriguez** [reply]
Hola, 

Yo utilicé el elemento summary.
Solo no logré darle el estilo para que la flechita se viera en la esquina del recuadro, pero de ahí en fuera lo puedes dejar igual.
[Ejemplo de W3Schools](https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_summary)

**Reply by OmarVenturaP** [reply]
les comparto mi código de JS, lo hice de esta forma.

![js](https://user-images.githubusercontent.com/99059639/157570218-e66e2140-f83f-4a98-919a-e41c07bff953.png)

**Comment by SujeiRodriguez** [comment]
ustedes saben como hacer redonda la sombra de los botones?, veo que al ponerla se expande a los lados, pero en la imagen se ve que es como hacia adentro

intente cambiando de varias formas los valores pero no pude 

espero darme a entender, perdonen

**Reply by AngelCruzO** [reply]
Busca generadores de shadow-box, ya que tengas el efecto solo coperas el código a tu archivo css

**Reply by SujeiRodriguez** [reply]
gracias :D

**Reply by AngelCruzO** [reply]
De nada, ya en esta época existen generadores para muchos de los atributos css, solo es buscar lo que necesitas

**Reply by SujeiRodriguez** [reply]
ahora entiendo como hacen las cosas tan rápido XD

**Reply by AngelCruzO** [reply]
Aparte hay librerías como boostrap, que ya traen clases que estructuran más la maquetación

**Reply by SujeiRodriguez** [reply]
eso también tengo pendiente de checar, pero me la paso haciendo otras cosas, en la madrugada me la pase checando el código de una pag donde hice una compra y me di cuenta que se aloja en otro dominio -______-, espero no vayan a estafar a mi tía con su compra :'v

**Reply by OmarVenturaP** [reply]
En lo particular yo lo hice con puro código CSS. No use ninguna librería, pero si es de paciencia. De todas maneras como dice el compañero, usando generadores se puede.

En lo particular solo hice generando sombra derecha y abajo de 2px y grosor de 5px.

**Reply by SujeiRodriguez** [reply]
la vdd es que no encontré bien el código en la pag haha pero me di una idea de los valores que tenia que mover,
para el botón azul de arriba deje
box-shadow: 0px 18px 13px -10px rgba(49, 49, 255, 0.397); y si queda bien

**Reply by SujeiRodriguez** [reply]
mmmm estoy pensando, que tal si lo hubiera encontrado con el f12... O__o

**Reply by OmarVenturaP** [reply]
El objetivo de esta práctica es que juegues con las sombras y profundidades. Con live server vas viendo en tiempo real y allí ya te vas dando una idea de cómo queda. Ajustas lo necesario. Creeme que me el caso de las sombras yo jugué un poco y no todos los elementos tienen los mismos parámetros

**Reply by AngelCruzO** [reply]
@OmarVenturaP Muchas felicidades por hacerlo desde cero, es un trabajo super lavorioso; pero la satisfaccion de hacerlo asi nadie te lo quita

**Reply by OmarVenturaP** [reply]
Gracias @AngelCruzO

**Comment by axlgoze** [comment]
Wow !! 😲una duda como le hiciste para poner los triangulos grises del fondo, son imagenes o los hiciste con CSS ? 😅

**Reply by OmarVenturaP** [reply]
La verdad los fondos si los hice utilizando un editor de imágenes.

 De la misma imagen que nos dió la landing page. Allí recorte nada más esos cuadros y se las ajuste en la página.

**Reply by OmarVenturaP** [reply]
Solo que si las hice por sección.

**Reply by axlgoze** [reply]
Gracias !

**Comment by GloriaCoralCerecedo** [comment]
Hola, ¿Cómo supiste los tipo de letra?

**Reply by OmarVenturaP** [reply]
No es igual, pero en Font Google encontré una similar. No tengo el dato de cuál es ahorita, pero al ratito te comento cuáles fueron las que ocupé.

**Reply by OmarVenturaP** [reply]
La tipografia que ocupe fueron estas: 

- Shippori Antique B1
- Rubik.

**Reply by GloriaCoralCerecedo** [reply]
Muchas gracias

**Comment by GloriaCoralCerecedo** [comment]
¿Las imágenes donde las buscaste? De la segunda imagen la encontré con fondo amarillo xd

**Reply by AngelCruzO** [reply]
No se si canva en su versión gratuita, te deje quitar fondos

**Reply by GloriaCoralCerecedo** [reply]
Creo que tengo la pro

**Reply by AngelCruzO** [reply]
Ponle editar la imagen y le quitas el fondo con canva, asi le hice yo

**Reply by GloriaCoralCerecedo** [reply]
Gracias, para colocar el color de fondo correcto abria que poner como una cuadro y poner la foro delante

**Reply by AngelCruzO** [reply]
puedes ponerle el fondo desde canva o ponerlo con css. con css yo utilice un div para ponerle el fondo degradado y dentro la img

**Reply by GloriaCoralCerecedo** [reply]
Eso iba a hacer gracias

**Reply by AngelCruzO** [reply]
de nada

**Comment by RodolfoMorquecho** [comment]
En la parte de la barra de navegación para la división blanca-azul cielo utilizaste una imagen? o con puro css? disculpa la pregunta tan basica.

**Reply by OmarVenturaP** [reply]
en la barra de navegación use un background blanco, pero le puse un poco de opacidad, ya a la barra de navegación le puse una "position: fixed", por eso se ve sobrepuesta y lo que le da el efecto de azul es la misma imagen que está en el fondo.

**Comment by GloriaCoralCerecedo** [comment]
Hola, el background que va como fondo de la página, ¿Dónde lo encontraste?

**Reply by OmarVenturaP** [reply]
Hola, lo hice con la misma imagen que compartió el MC

**Reply by GloriaCoralCerecedo** [reply]
Oh, no entiendo bien disculpa

**Reply by hackit92** [reply]
Te recomiendo hacerla en svg, es mucho más útil al momento de hacer webs y no es complicado =)

**Reply by GloriaCoralCerecedo** [reply]
Gracias
